assets/app/styles/_core.less

```diff
@@ -832,6 +832,60 @@ a.disabled-link {
   }
 }
 
+.edit-form {
+  margin: 30px auto;
+  .form-group {
+    .multiselector {
+      width: 32%;
+      display: inline-block;
+    }
+    .simpleselector {
+      display: inline-block;
+    }
+    .multiselector-label {
+      display: none;
```

Comment:
I've used those classes before, but for some reason (to me unknown)  I forgot about it. Thanks :+1: